contents/blog/the-truth-about-tiger-teams.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Why small teams crush tiger teams
+date: 2025-12-04
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+    - natalia-amorim
+featuredImage: >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/hog_ql.png
+featuredImageType: full
+category: General
+tags:
+    - Culture
+    - Product
+seo:
+    metaTitle: 'Why small teams crush tiger teams'
+    metaDescription: "Tiger teams sound great in theory, but they rarely deliver. Here's why small teams beat tiger teams every time, and how to structure your org for speed."
+---
+There are a few core milestones in the lifecycle of a PostHog employee. 
+
+The first one is when your friends or family ask, "So… what does your company actually do?" and suddenly you're explaining the concept of B2B SaaS to your 87-year-old grandpa who just recently figured out how to operate Alexa.
+
+Once that's settled, one of two things happens: you've either bored them to death and they never ask about it again, or they double down on wanting to better understand how on earth you do what you do.
+
+"But how do you handle [that many products](/products)?!" 
+
+Congratulations, you've reached the next milestone: talking about [small teams](/small-teams). If they've ever worked in a large company, the response is predictable: "Oh okay, so like tiger teams."
```

Comment:
I think in general, I would plug other content before handbook or other pages (other than product or pricing pages).

It would be good to swap this one in here: https://posthog.com/newsletter/small-teams